aldryn_events/tests/frontend/protractor.conf.js

```diff
@@ -20,14 +21,48 @@ var config = {
     },
 
     onPrepare: function () {
+        // Set browser window size
+        aldrynTestsLibrary.setWindowSize(browser.params.browserConfig.width,
+            browser.params.browserConfig.height);
+        // Set Angular site flag
         browser.ignoreSynchronization = true;
+        // Pre-create folders for reporting
+        aldrynTestsLibrary.preCreateFolder(
+            'aldryn_events/tests/frontend/integration/screenshots');
+        aldrynTestsLibrary.preCreateFolder(
+            'aldryn_events/tests/frontend/integration/reports');
+        // Automatically store a screenshot at the end of each test
+        aldrynTestsLibrary.storeScreenshot(
+            'aldryn_events/tests/frontend/integration/screenshots/');
     },
 
-    // Options to be passed to Jasmine-node.
+    // Name of the process executing this capability.  Not used directly by
+    // protractor or the browser, but instead pass directly to third parties
+    // like SauceLabs as the name of the job running this test
+    name: 'aldryn-events',
+
+    // Params for setting browser window width and height - can be also
+    // changed via the command line as: --params.browserConfig.width 1024
+    params: {
```

Comment:
why do we need to define a size? it should be universal applicable as we do not know the size on various devices (we also test mobile cases with SauceLabs)